src/shipit_signoff/docs/architecture.gv

```diff
@@ -0,0 +1,21 @@
+digraph {
+	shipit_signoffs [label="Signoffs Service" shape=box]
+		shipit_signoffs -> db
+		shipit_signoffs -> balrog_admin_api
+		shipit_signoffs -> shipit_notifications
+	shipit_pipeline [label="Pipeline Service" shape=box]
+		shipit_pipeline -> shipit_signoffs
+	shipit_frontend [label="Ship It UI" shape=box]
+		shipit_frontend -> shipit_pipeline
+		shipit_frontend -> shipit_signoffs
+	user [label=User shape=box]
+		user -> shipit_frontend
+		user -> auth0
+		user -> "login.taskcluster.net"
+	shipit_notifications [label="Notifications Service" shape=box]
+	db [label=db shape=box]
+	balrog_admin_api [label="Balrog Admin API" shape=box]
+	auth0 [label=Auth0 shape=box]
+		auth0 -> shipit_signoffs
+	"login.taskcluster.net" [label="login.taskcluster.net" shape=box]
+}
```

Comment:
This first diagram is just a basic quick thing, we can certainly do better. I'll have a look at the sphinx extension, though.